src/compat/libc/stdlib/wctomb.c

```diff
@@ -0,0 +1,28 @@
+/**
+ * @file
+ * @brief wctomb function.
+ *
+ * @see stdlib.h
+ *
+ * @date 07.12.18
+ * @author Chubukov Filipp
+ */
+
+#include <stdlib.h>
+#include <string.h>
+#include <defines/size_t.h>
+#include <defines/wchar_t.h>
+
+int wctomb(char *out,  const wchar_t *in) {
```

Comment:
Same problem as `mbtowc`/`mbstowcs`: this function is not the same with `wcstombs`

Reply:
change function